curriculum/challenges/english/08-coding-interview-prep/project-euler/problem-99-largest-exponential.english.md

```diff
@@ -7,10 +7,15 @@ forumTopicId: 302216
 
 ## Description
 <section id='description'>
-Comparing two numbers written in index form like 211 and 37 is not difficult, as any calculator would confirm that 211 = 2048 < 37 = 2187.
-However, confirming that 632382518061 > 519432525806 would be much more difficult, as both numbers contain over three million digits.
-Using base_exp.txt (right click and 'Save Link/Target As...'), a 22K text file containing one thousand lines with a base/exponent pair on each line, determine which line number has the greatest numerical value.
-NOTE: The first two lines in the file represent the numbers in the example given above.
+
+Comparing two numbers written in index form like 2<sup>11</sup> and 3<sup>7</sup> is not difficult, as any calculator would confirm that 2<sup>11</sup> = 2048 < 3<sup>7</sup> = 2187.
+
+However, confirming that 632382<sup>518061</sup> > 519432<sup>525806</sup> would be much more difficult, as both numbers contain over three million digits.
+
+Using `baseExp`, a 2D array containing one thousand inner arrays with a base/exponent pair, determine which pair has the greatest numerical value.
```

Comment:
Same again.

Reply:
Same as above, this one could read:

"Using the included 2D array of base/exponent pairs, determine which pair has the greatest numerical value."